lib/streamlit/type_util.py

```diff
@@ -97,18 +98,30 @@
     "trigger_value",
 ]
 
+V_co = TypeVar("V_co", covariant=True)
```

Comment:
Small ask: It would be nice to include a link to the [PEP 484 section on covariance and contravariance](https://peps.python.org/pep-0484/#covariance-and-contravariance) here (which itself links to more thorough introductions to the concepts).

I'd guess that a good number of people that interact directly with the codebase aren't too familiar with the concepts, so having a pointer to useful resources here would be nice when people inevitably dig through the code to try to figure out what `V_co` means.

Reply:
Good point. Added a comment as you suggested. It is a somewhat inaccessible topic, no doubt about that.